lib/workos/api.ex

```diff
@@ -8,9 +8,11 @@ defmodule WorkOS.API do
   """
   def client(opts \\ []) do
     auth = WorkOS.api_key(opts)
-    if is_map_key(opts, :access_token) do
-      auth = opts[:access_token]
+
+    if opts[:access_token] do
+      ^auth = opts[:access_token]
```

Comment:
Is there a wrong assertion here, that access_token should be the same as API key?
This seems to be breaking the `SSO.get_profile` function.

@jacobiajohnson @blairlunceford